apis/Google.Cloud.Firestore/Google.Cloud.Firestore/Query.cs

```diff
@@ -45,6 +45,9 @@ public class Query : IEquatable<Query>
         private readonly IReadOnlyList<FieldPath> _projections; // May be null
         private readonly Cursor _startAt;
         private readonly Cursor _endAt;
+
+        // we should never allow a new query to be created for a different root,
```

Comment:
Nit: capital W.

(Having worked with codebases that don't use full sentences with appropriate punctuation and spacing for comments, I've learned that it's really distracting - at least for me. Hence nits that many other folks would be perfectly happy to let go...)